### PR TITLE
Ink! 4.0 update

### PR DESCRIPTION
**Pull request by PierreOssun**
- removed rmrk chain extension
- updated to ink! 4.0 rc
- only compatible with node with polkadot v0.9.36
- readme will be updated after igration to swanky dapps (and this repo will be made private)

### CODE REVIEW

File: tests/assets.spec.ts [hunk 1]
```diff
@@ -90,10 +96,10 @@ describe('ASSETS', () => {
         await assets.tx.mint(ASSET_ID, alice.address,1000,{gasLimit: gas2 });
 
         // @ts-ignore
```
Comment (Maar-io) [inline tests/assets.spec.ts:98]: can all of these` ts-ignore` be moved in typescript congif file?

File: tests/assets.spec.ts [hunk 2]
```diff
@@ -109,10 +115,10 @@ describe('ASSETS', () => {
         await assets.tx.approveTransfer(ASSET_ID, bob.address, 100, {gasLimit: gas3 });
 
         // @ts-ignore
-        await expect((await assets.query.allowance(ASSET_ID, alice.address, bob.address)).value.toNumber()).to.equal(100)
+        await expect((await assets.query.allowance(ASSET_ID, alice.address, bob.address)).value.unwrap().toNumber()).to.equal(100)
```
Comment (Maar-io) [inline tests/assets.spec.ts:118]: can you use` ?` instead of `unwrap()`? Just asking

**Review by Maar-io — APPROVED** [review]
LGTM

**Review by niklabh — APPROVED** [review]

File: tests/assets.spec.ts (hunk 2 above)
Comment (ashutoshvarma) [reply tests/assets.spec.ts:118]: Typescript don't have rust like escape '?' operator.

**Pull request merged**